### DISCUSSION TITLE
I am doing a presentation at ApacheCon Asia on "The Art of Technical Writing" and wish to use your wiki as an exmaple

### DISCUSSION THREAD

**Discussion by msacks-dev** [requarks/wiki#5448] [General]
I am doing at presentation at ApacheCon Asia 2022 called "The Art of (Software) Technical Writing" and I'd love to speak to some of the dev team and contributors about this wiki. I think it's great software and want to use it as an example  If you are interested in hearing my talk or seeing the contents of my talk feel free to message me or email me matthew at mdsitservices.com.

**Comment by julianalsemmani** [comment]
It will probably be easier to get in touch with the developer if you go to their profile, and see if they have a website or email available.

Or you can join the Wiki.js discord or slack, and contact them there! :)
